scripts/ci-entrypoint.sh

```diff
@@ -212,6 +212,7 @@ if [[ -n "${TEST_CCM:-}" ]]; then
     fi
 
     export CCM_LOG_VERBOSITY="${CCM_LOG_VERBOSITY:-4}"
+    export CCM_CLUSTER_CIDR="${CCM_CLUSTER_CIDR:-10.244.0.0/16}"
```

Comment:
where is this default value coming from? Is there any way we can determine it programmatically instead of hardcoding a default?

Reply:
Also what about dual stack clusters that have two cidrs?

Reply:
The default value is in the readme file and I don't find a good way to avoid hardcoding? https://github.com/kubernetes-sigs/cloud-provider-azure/blob/c8640b1bee6022318753bde033c8eb4d3089ebe7/helm/cloud-provider-azure/README.md#configurable-values 

If dualstack, then users should specify the cluster cidr. And this is the purpose of my change.

Reply:
@CecileRobertMichon sorry, I misunderstood your question. Yes, cluster config sounds reasonable and I think we should use the `cidrBlocks` values. Is the "not look at" a typo or you suggest some other fields?

`cloudControllerManager.clusterCIDR` of ccm can take 2 values, e.g. "10.244.0.0/16\,2001:1234:5678:9a40::/64"